### DISCUSSION TITLE
关于UDP转发使用

### DISCUSSION THREAD

**Discussion by blue1314520** [natfrp/wiki#195] [使用问题]
网站上只留下了远程端口和本地端口一致的解决方法，事实上依然无法解决问题，是服务器针对UDP转发做了限制？其他FRP是直接能用的，是基于UDP转发流量消耗较少，真正未开放UDP转发造成的么，盼回复

**Comment by Sawaumi** [comment]
附议，UDP隧道宣布启动但服务无法连接

**Comment by fakeboboliu** [comment]
并无部署限制。

特别的，成都和辽宁节点无法使用 UDP 系提供商做出的限制，
允许在这两类节点上创建 UDP 隧道应为 Bug，已经提交维修。
